### ISSUE TITLE
In side docs, cannot activate simulator or expanded snippets when tabbing through

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2343]
**Describe the bug**

the buttons at the bottom are not tab accessible - they get highlighted, but don't activate on space or enter being pressed

**To Reproduce**
1. open a project, right click on ``on start``, and select help to open siide docs
2. tab through
3. when the simulator button attached to the bottom of a snippet is selected, try activate

**Expected behavior**

should capture space / enter button press and expand

**Screenshots**

![2019-08-02 15 52 14](https://user-images.githubusercontent.com/5615930/62402739-0059dd80-b53e-11e9-8ce2-75f0769f6ef2.gif)

**Comment by jwunderl** [comment]
This is a duplicate of https://github.com/microsoft/pxt-microbit/issues/948
